src/wasm.rs

```diff
@@ -1,11 +1,13 @@
 #![allow(clippy::panic)]
 
+#[cfg(feature = "fluxlang")]
 mod lang;
 mod lsp;
 #[cfg(test)]
 mod tests;
 
 pub use self::lang::Flux;
+#[cfg(feature = "fluxlang")]
```

Comment:
```suggestion
#[cfg(feature = "fluxlang")]
pub use self::lang::Flux;
```
Should these two lines get switched?

Reply:
Yeah, that must've been an editor hiccup. Luckily, it caused a failing test.